bash/install.sh

```diff
@@ -1,29 +1,30 @@
 #!/bin/bash
```

Comment:
I have release v1.0.0 and it changes both the install and update scripts. These changes are no longer needed.